clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11979,6 +11979,9 @@ def warn_ivdep_attribute_argument : Warning<
 def warn_attribute_spelling_deprecated : Warning<
   "attribute %0 is deprecated">,
   InGroup<DeprecatedAttributes>;
+def warn_attribute_deprecated_ignored : Warning<
+  "attribute %0 is deprecated; attribute ignored">,
```

Comment:
We could re-use `warn_attribute_ignored` instead, but that would be two warnings for one attribute, so I don't have a strong opinion here.

Reply:
The spec authors preferred a single warning. So I think this shoul dbe kept as-is